### PR TITLE
Add 'Computer' faker

### PR DESCRIPTION
**Pull request by panilya**

### CODE REVIEW

File: src/main/java/net/datafaker/Computer.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    public String platform() {
+        return faker.resolve("computer.platform");
+    }
+
+    public String operatingSystem() {
+        List<String> operatingSystems = new ArrayList<>();
+        operatingSystems.add("linux");
+        operatingSystems.add("macos");
+        operatingSystems.add("windows");
+        return faker.resolve("computer.os." + operatingSystems.get(faker.random().nextInt(0, 2)));
```
Comment (panilya) [inline src/main/java/net/datafaker/Computer.java:28]: @bodiam Is there a better way to implement this?

Comment (snuyanzin) [reply src/main/java/net/datafaker/Computer.java:28]:
```java
return faker.resolve("computer.os." + faker.options().option("linux", "macos", "windows"));
```

**Comment by bodiam** [comment]
Nice one!! Thanks!

**Comment by bodiam** [comment]
(Sorry, I passed the wrong button!)

File: src/main/java/net/datafaker/Computer.java (hunk 1 above)
Comment (bodiam) [reply src/main/java/net/datafaker/Computer.java]: You could also use an Enum here

**Pull request merged**